pumpkindb_engine/src/script/macros.rs

```diff
@@ -243,6 +243,8 @@ macro_rules! alloc_and_write {
     }};
 }
 
+// TODO: use or remove?
+#[allow(unused_macros)]
```

Comment:
I think it's a good idea to keep the `unused_macros` attributes there. They do help silencing these irrelevant warnings.